.travis.yml

```diff
@@ -40,9 +40,9 @@ after_success:
   - echo "NPM_TAG=$NPM_TAG"
   - >
     [ "${TRAVIS_BRANCH}" = "master" ] &&
-    npm version $GIT_TAG -m "Version $GIT_TAG built by Travis CI - https://travis-ci.com/$TRAVIS_REPO_SLUG/builds/$TRAVIS_JOB_ID" &&
+    yarn version --no-git-tag-version --new-version $GIT_TAG &&
```

Comment:
@nlaz you can also ask NPM directly for which versions are available, although it's not quite as descriptive as GitHub's 'releases' list

```sh
$ npm view meetup-web-platform versions
```

To specifically find the latest beta version, use

```
$ npm view meetup-web-platform@beta version
1.0.711-beta
```